spec/integration/collaborators_spec.js

```
@@ -0,0 +1,155 @@
+const Wiki = require("../../src/db/models").Wiki;
+const User = require("../../src/db/models").User;
+const Collaborator = require("../../src/db/models").Collaborator;
+const sequelize = require("../../src/db/models/index").sequelize;
+const request = require("request");
+const base = "http://localhost:8000/wikis/";
+const accountBase = "http://localhost:8000/account/";
+const server = require("../../src/server");
+
+describe("routes : collaborators", () => {
+
+    beforeEach((done) => {
+        this.premiumUser;
+        this.privateWiki;
+
+        sequelize.sync({force: true}).then(() => {
+    
+            User.create({
+              name: "Patty Premium",
+              email: "premiumUser@example.com",
+              password: "1333324"
+            }).then((user) => {
+              User.findOne({where: {id: user.id}}).then(() => {
+    
+                request.post(`${accountBase}upgrade`, (err, res, body) => {
+                  User.findOne({where: {id: user.id}}).then((premiumdUser) => {
+                    this.premiumUser = premiumdUser;
+                    
+                    Wiki.create({
+                        title: "Wizard of Oz",
+                        body: "Everything is better in green!",
+                        userId: this.premiumUser.id,
+                    }).then((wiki) => {
+                        this.privateWiki = wiki;
+                    }).catch((err) => {
+                        console.log(err);
```

Throughout this whole file you're swallowing exceptions (catching them, logging them, and then proceeding on your merry way like nothing happened). You almost never want to do that IRL. If you can't meanigfully handle an error then the correct response is usually to let it crash. This is doubly true for testing code, which should definitely fail on unanticipated exceptions.